src/avi/layout/panes.clj

```diff
@@ -235,10 +235,15 @@
   :ret ::split)
 (defn split-pane
   [editor new-lens direction]
-  (+> editor
-    (split-pane' new-lens direction)
-    simplify-panes
-    resize-panes))
+  (let [{:keys [::lens] [[_] [rows cols]] :avi.layout/shape :as pane}
+          (current-pane editor)]
+    (if (or (and (= direction :horizontal) (> (/ rows 2) 2))
```

Comment:
I'm happy to keep this - but it isn't sufficient since it's possible to resize the terminal window after you create a split.  Not sure what to do in this case.